Sources/XMTP/Codecs/ReplyCodec.swift

```diff
@@ -66,4 +66,8 @@ public struct ReplyCodec: ContentCodec {
     public func fallback(content: Reply) throws -> String? {
         return "Replied with “\(content.content)” to an earlier message"
     }
+	
+	public func shouldPush(content: Reply) throws -> Bool {
+		return false
```

Comment:
Do we not want to push for replies?

Reply:
the scope of the task only includes `textCodec` and `Composite`